omim2obo/utils/utils.py

```diff
@@ -30,10 +30,10 @@ def get_d2g_digenic_protections(path=DISEASE_GENE_PROTECTED_PATH) -> Dict[Tuple[
     :return: Dictionary with phenotype and gene MIMs as keys and ORCID of curator as values.
     """
     df = pd.read_csv(path, sep='\t').fillna('')
-    for col in ['phenotype_mim', 'gene_mim']:
+    for col in ['phenotype_mim', 'gene_mim', 'hgnc_id']:
         df[col] = df[col].apply(lambda x: x.split(':')[1])
     return {
-        (x['phenotype_mim'], x['gene_mim']): ORCID[x['orcid']] if x['orcid'] else None
+        (x['phenotype_mim'], x['gene_mim'], x['hgnc_id']): ORCID[x['orcid']] if x['orcid'] else None
```

Comment:
Return the hgnc_id since the omim gene to hgnc gene relationship needs to be added in omim.ttl for entries in "protected-disease-gene.tsv" regardless of whether the OMIM file has the omim gene to hgnc gene relationship mapping.